theseus/embodied/kinematics/robot_model.py

```diff
@@ -4,10 +4,12 @@
 # LICENSE file in the root directory of this source tree.
 
 import abc
+from typing import Dict
 
 import torch
+import differentiable_robot_model as drm
 
-from theseus.geometry import SE2, LieGroup, Point2, Vector
+from theseus.geometry import SE2, SE3, LieGroup, Point2, Vector
 
 
 class RobotModel(abc.ABC):
```

Comment:
We could rename this to something more general so we can derive from it urdf robot models and additionally later, human models, hand models, etc.